src/components/dialogs/network-modifications/by-filter/by-assignment/assignment/assignment-constants.ts

```diff
@@ -292,6 +292,16 @@ export const FIELD_OPTIONS = {
         dataType: DataType.ENUM,
         values: LOAD_TYPES,
     },
+    OPERATIONAL_LIMITS_GROUP_1: {
```

Comment:
Like in commons-ui I think it should be precised that those are the selected/active operational limits groups, as opposed to the the regular ones.
```suggestion
   SELECTED_OPERATIONAL_LIMITS_GROUP_1: {
```